msdk-io/msdk-io-mzml2/src/main/java/io/github/msdk/io/mzml2/MzMLFileWriter.java

```diff
@@ -0,0 +1,562 @@
+/*
+ * (C) Copyright 2015-2017 by MSDK Development Team
+ *
+ * This software is dual-licensed under either
+ *
+ * (a) the terms of the GNU Lesser General Public License version 2.1 as published by the Free
+ * Software Foundation
+ *
+ * or (per the licensee's choosing)
+ *
+ * (b) the terms of the Eclipse Public License v1.0 as published by the Eclipse Foundation.
+ */
+
+package io.github.msdk.io.mzml2;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.util.List;
+import java.util.Optional;
+
+import javax.annotation.Nonnull;
+import javax.xml.stream.XMLOutputFactory;
+import javax.xml.stream.XMLStreamException;
+import javax.xml.stream.XMLStreamWriter;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.sun.xml.txw2.output.IndentingXMLStreamWriter;
+
+import io.github.msdk.MSDKException;
+import io.github.msdk.MSDKMethod;
+import io.github.msdk.datamodel.chromatograms.Chromatogram;
+import io.github.msdk.datamodel.msspectra.MsSpectrumType;
+import io.github.msdk.datamodel.rawdata.ActivationInfo;
+import io.github.msdk.datamodel.rawdata.IsolationInfo;
+import io.github.msdk.datamodel.rawdata.MsScan;
+import io.github.msdk.datamodel.rawdata.PolarityType;
+import io.github.msdk.datamodel.rawdata.RawDataFile;
+import io.github.msdk.io.mzml2.data.MzMLArrayType;
+import io.github.msdk.io.mzml2.data.MzMLBitLength;
+import io.github.msdk.io.mzml2.data.MzMLCV;
+import io.github.msdk.io.mzml2.data.MzMLCVParam;
+import io.github.msdk.io.mzml2.data.MzMLCompressionType;
+import io.github.msdk.io.mzml2.util.MzMLPeaksEncoder;
+
+public class MzMLFileWriter implements MSDKMethod<Void> {
+
+  private static final String dataProcessingId = "MSDK_mzml_export";
+  private static final String softwareId = "MSDK";
+  private static final String XML_ENCODING = "UTF-8";
+  private static final String XML_VERSION = "1.0";
+  private static final String MZML_NAMESPACE = "http://psi.hupo.org/ms/mzml";
+  private static final String DEFAULT_VERSION = "1.1.0";
+  private static final String CV_REF_MS = "MS";
+
+  private static final String TAG_MZML = "mzML";
```

Comment:
why not put all these constants in into a holder class like MzMLTags? Use can refer to this class in both the import and the export modules?